From 717d1edce85404d4b42d33bef1df8afb81ec6201 Mon Sep 17 00:00:00 2001
From: SferaDev <sferadev@gmail.com>
Date: Mon, 28 Oct 2013 16:52:34 +0100
Subject: [PATCH 1/2] system_extras: Add make_ext4fs utility

Add make_ext4fs into $OUT/Utilities needed for some devices like Galaxy S I9000

Change-Id: Id9c9ce79282859542a1e44501b19594145d80275
---
 ext4_utils/Android.mk | 19 +++++++++++++++++++
 1 file changed, 19 insertions(+)

diff --git a/ext4_utils/Android.mk b/ext4_utils/Android.mk
index ad87997..c99cc28 100644
--- a/ext4_utils/Android.mk
+++ b/ext4_utils/Android.mk
@@ -49,6 +49,25 @@ endif
 include $(BUILD_HOST_EXECUTABLE)
 
 
+include $(CLEAR_VARS)
+LOCAL_SRC_FILES := make_ext4fs_main.c
+LOCAL_MODULE := utility_make_ext4fs
+LOCAL_MODULE_STEM := make_ext4fs
+LOCAL_MODULE_TAGS := optional
+LOCAL_FORCE_STATIC_EXECUTABLE := true
+LOCAL_MODULE_CLASS := UTILITY_EXECUTABLES
+LOCAL_MODULE_PATH := $(PRODUCT_OUT)/utilities
+LOCAL_UNSTRIPPED_PATH := $(PRODUCT_OUT)/symbols/utilities
+LOCAL_STATIC_LIBRARIES += \
+    libext4_utils_static \
+    libselinux \
+    libsparse_static \
+    libz \
+    libcutils \
+    libc
+
+include $(BUILD_EXECUTABLE)
+
 #
 # -- All host/targets excluding windows
 #
-- 
1.9.1

